sematic/cli/clean.py

```diff
@@ -0,0 +1,78 @@
+# Standard Library
+import logging
+import os
+from logging.config import dictConfig
+
+# Third-party
+import click
+
+# Sematic
+from sematic.api.endpoints.auth import get_cleaner_api_key
+
+# TODO: will use API client to perform cleaning operations
+# import sematic.api_client as api_client
+from sematic.cli.cli import cli
+from sematic.config.user_settings import UserSettingsVar
+from sematic.logging import make_log_config
+
+logger = logging.getLogger(__name__)
+
+
+@cli.command("clean", short_help="Clean up orphaned objects")
```

Comment:
Please comment this out until the contents is implemented, so that this option does not show up in the CLI help.
Is it necessary to merge this PR before finishing the implementation?

Reply:
I can wait to merge this until the options are implemented. This PR is here first so I can get feedback on the overall approach

Reply:
Enabled now!